JS/models/ExperimentalTypes.js

```diff
@@ -1,8 +1,8 @@
 const ExperimentalTypes   = {
```

Comment:
Please use CAPITAL_LETTER notation for constants naming.

Reply:
All constants in capital letters